addons/sale/models/res_company.py

```diff
@@ -47,6 +47,29 @@ class ResCompany(models.Model):
         ],
         string="Sale onboarding selected payment method")
 
+    property_downpayment_account_id = fields.Many2one(
```

Comment:
Don't name it `property` please, it's not a company_dependent field, it's a simple company field.